content/updates/04-2020/2-associations-updates.md

```diff
@@ -0,0 +1,30 @@
+# Product associations update
+
+
+## View the number of associations per product
+::: meta-data type="New" features="Product associations, Productivity, Data Governance" available="10th of April" link-to-doc="../articles/products-associations.html#view-the-products-associations"
+:::
+
+We now display the **total number of associations per product or product model nearby the `Associations` tab** when you are your product form.    
+You can now quickly see the number of associations for your products.
+
+We also show you the **total number of associations per association type**, when you are on your product associations page.
+
+![Associations number](../img/associations-number.png)
+
+## Easy access to associations
+::: meta-data type="New" features="Product associations, Productivity, Data Governance" available="10th of April" link-to-doc="../articles/products-associations.html#open-an-associated-product"
+:::
+
+Many of you were asking for it... From the `Associations` tab, you can now click on an associated product or product model and have it displayed in a new tab!
+
+## Association screen improvement
+::: meta-data type="New" features="Product associations, Productivity" available="10th of April" link-to-doc="../articles/what-about-measurements.html"
```

Comment:
```suggestion
::: meta-data type="New" features="Product associations, Productivity" available="mid-April" link-to-doc="../articles/what-about-measurements.html"
```

Reply:
The link is wrong.

Reply:
Appropriate link: "articles/products-associations.html"

Reply:
f*ck - I think I copied the link but not edit them :/ will change it thank you.